packages/plugins/graphql-plugin-auth/src/Neo4jGraphQLAuthJWTPlugin.ts

```diff
@@ -62,7 +75,7 @@ class Neo4jGraphQLAuthJWTPlugin {
 
                 result = jsonwebtoken.decode(token, { json: true }) as unknown as T;
             }
-
+            //In case that the secret is null, make sure the 'tryToResolveKeys' is ran before the decode method.
```

Comment:
A similar text could be used as a `debug()` output if `this.secret === null` I suppose? Along the line of what you did in the packages/plugins/graphql-plugin-auth/src/Neo4jGraphQLAuthJWKSPlugin.ts file.

Reply:
resolved by 90fc0a0c7fa7c4fdeeacfb1ed13b4eaef98853fd

Reply:
btw, the `debug.enabled` is always false so on the plugins the `debug()` would not do anything, would it? I found it out because I wanted to create a test case when `secret is null` to check if debug is called or `console.log` but it never has an output unless I set the `debug.enabled = true`